src/test/java/org/tarantool/TarantoolControl.java

```diff
@@ -216,44 +193,67 @@ public void waitStarted(String instanceName) {
     }
 
     /**
-     * Wait until the instance will be stopped.
+     * Sends <code>stop</code> command using tarantoolctl.
      *
-     * Use tarantoolctl status instanceName.
+     * @param instanceName target instance name
      */
-    public void waitStopped(String instanceName) {
-        while (status(instanceName) != 1) {
-            sleep();
+    public void stop(String instanceName) {
+        executeControlCommand("stop", instanceName);
+    }
+
+    /**
+     * Sends <code>stop</code> command using tarantoolctl
+     * and blocks until the process will be down.
+     *
+     * @param instanceName target instance name
+     */
+    public void stopAndAwait(String instanceName) {
+        ProcessCommand fetchInstancePidCommand =
+            new ProcessCommand(buildInstanceEnvironment(instanceName), "cat", instanceName + ".pid");
```

Comment:
Doing read() via fork() + exec() feels badly. I understand, it is not a performance critical place. But… I don't know. It is not the right thing.